packages/mc-html-template/lib/load-env.js

```diff
@@ -1,16 +1,93 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
 const fs = require('fs');
+const replaceall = require('replaceall');
 
 // Keep a reference to the loaded config so that requiring the module
 // again will result in returning the cached value.
 let loadedEnv;
 
-module.exports = configPath => {
-  if (loadedEnv) return loadedEnv;
+/**
+ * NOTE:
+ *  Allows env variable placeholders e.g. `${env:MC_API_URL}`.
+ *  Other placeholder types might be supported in the future.
+ */
+const variableSyntax = RegExp('\\${([ ~:a-zA-Z0-9._\'",\\-\\/\\(\\)]+?)}', 'g');
+const envRefSyntax = RegExp(/^env:/g);
+
+const hasVariablePlaceholder = valueOfEnvConfig =>
+  typeof valueOfEnvConfig === 'string' &&
+  valueOfEnvConfig.match(variableSyntax);
+const isEnvVariablePlaceholder = valueOfPlaceholder =>
+  valueOfPlaceholder.match(envRefSyntax);
+const substituteEnvVariablePlaceholder = (
+  valueOfPlaceholder,
+  matchedString,
+  valueOfEnvConfig
+) => {
+  const requestedEnvVar = valueOfPlaceholder.split(':')[1];
+  const valueOfEnv = process.env[requestedEnvVar];
+
+  if (!valueOfEnv) {
+    throw new Error(
+      `Missing '${requestedEnvVar}' specified in config as 'env:${requestedEnvVar}'.`
+    );
+  }
+
+  const substitutedEnvConfig = replaceall(
```

Comment:
Yeah, it could work without. However the library is 10 lines and helps with the auto-escaping too in the string. That's why I felt it's useful. I you want to remove it feel free to add a commit.

Reply:
Yeah but I would avoid depending on a library for something simple like this.

https://github.com/commercetools/merchant-center-application-kit/pull/1380/commits/ff70e05165159cf9a7ed3ef30e726cc100bcf5d5

Reply:
😎 all good.